dev/com.ibm.ws.jdbc_fat/fat/src/com/ibm/ws/jdbc/fat/tests/ConfigTest.java

```diff
@@ -294,7 +247,7 @@ public void testConfigChangeAuthData() throws Throwable {
         String originalUserName = testContainer.getUsername();
 
         // First check that authData matches with what was provided by the TestContainer
-        runInServlet(basicfat, "DataSourceTestServlet", "testConfigChangeAuthDataOriginalValue", "originalUsername=" + originalUserName);
+        runTestWithResponse(server, basicfat, "testConfigChangeAuthDataOriginalValue&originalUsername=" + originalUserName);
```

Comment:
What was behind the decision to the switch from runInServlet to runTestWithResponse?  Were you just wanting to eliminate redundant testing methods?

Reply:
Yes!  runTestWithResponse was available through the `componenttest.topology.utils` package.  I hadn't realized that before.